orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/monitoreddeploy/NotifyDeployCompletedTask.java

```diff
@@ -43,10 +44,18 @@
       Stage stage,
       MonitoredDeployStageData context,
       DeploymentMonitorDefinition monitorDefinition) {
-    // TODO(mvulfson): actually populate the request data
     DeploymentCompletedRequest request = new DeploymentCompletedRequest(stage);
-    monitorDefinition.getService().notifyCompleted(request);
 
+    Stage foundStage =
```

Comment:
this isn't super safe because you could get someone else's rollback, use `directAncestors` instead